### PR TITLE
Update Introduction

### PR DESCRIPTION
**Pull request by adnanmorsy97**
I have fixed some spelling and grammar mistakes

### CODE REVIEW

**Comment by adnanmorsy97** [comment]
@JohannesLichtenberger could you please merge it :) ?

File: documentation.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ title: SirixDB - Documentation
...
...ollowed by the removal of the outdated data. The latter may be some time later from a background process. Data, however, naturally evolves. It is often of great value to keep the history of our data. We, for instance, might record the payroll of an employee on the first of March in 2019. Let’s say it’s 5000€ / month. Then as of the fifteenth of April, we notice, that the recorded payroll was wrong and correct it to 5300€. Now, what’s the answer to what the salary was in March, first in 2019? Database Systems, which only preserve the most recent version, don’t even know that the payroll wasn’t right. Our answer to this question depends on what source we consider most authoritative: the record or reality? The fact that they disagree effectively splits this payroll event into two tracks of time, rendering neither source entirely accurate. Temporal database systems such as SirixDB help answer questions such as these easily. We provide at all times the transaction time, which SirixDB sets, once a transaction commits (when is a fact valid in the database / the record). Application or the valid time has to be set by the application itself (when is a fact valid in the real world/reality?).
```
Comment (mosheduminer) [inline documentation.md:12]: > Application or the valid time

This is incorrect. Perhaps "Application (also known as "valid") time

File: documentation.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ title: SirixDB - Documentation
...
+SirixDB is very space-efficient. Depending on the versioning algorithm, it only copies changed records plus possibly a few more during writes. Thus, SirixDB, for instance, usually does not copy a whole database page if only a single record has changed. Instead, SirixDB syncs page fragments to persistent storage during a commit. We can drop the requirement to cluster-related nodes physically. Sequentially accessing physically dispersed nodes on flash-based storage will be in the same order of magnitude as accessing physically clustered nodes on a disk. SirixDB never allows to override or delete old revisions. A single read-write transaction appends data at all times. For sure, you can revert a resource to a specific revision and commit changes based on this version. All revisions in-between will be accessible for data audits. Thus, SirixDB can support answering questions such as who changed what and when.
```
Comment (mosheduminer) [inline documentation.md:16]: > cluster-related

This is definitely meant to be "cluster related". Please change this back.

File: documentation.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ Data audits are about how specific records have changed. Time Travel queries can
...
+SirixDB is a storage system, which brings versioning to a sub-file granular level while taking full advantage of flash-based drives as SSDs. As such, per revision as well as per page deltas are stored. Time-complexity for retrieval of records and the storage are logarithmic (`O(log n)`). Space complexity is linear (`O(n)`). Currently, we provide several APIs which are layered. A very low-level page-API, which handles the storage and retrieval of records on a per page-fragment level. A transactional cursor-based API to store and navigate through records (currently XML as well as JSON nodes) on top. A DOM-alike node layer for simple in-memory processing of these nodes, which is used by Bracket, a sophisticated XQuery processor. And last but not least a RESTful asynchronous HTTP-API. SirixDB provides
```
Comment (mosheduminer) [inline documentation.md:25]: Brackit is the name of the XQuery processor, not Bracket. Please change this back.

File: documentation.md [hunk 4]
```diff
@@ -39,25 +39,26 @@ You may find a quick overview about the [main features](/features.html) useful.
 ### API Documentation
 We provide several APIs to interact with SirixDB.
 
-1. The [transactional cursor API](/transactional-cursor-api.html) is a powerful low-level API.
```
Comment (mosheduminer) [inline documentation.md:42]: It looks like the only change here is the removal the the link. Why remove the link? Please change this back.

File: documentation.md [hunk 5]
```diff
@@ -39,25 +39,26 @@ You may find a quick overview about the [main features](/features.html) useful.
...
 
 ### Publications
 Articles published on Baeldung:
 - [A Guide to SirixDB](https://www.baeldung.com/sirix)
 
 Articles published on Medium: 
-- [Asynchronous, Temporal  REST With Vert.x, Keycloak and Kotlin](https://medium.com/hackernoon/asynchronous-temporal-rest-with-vert-x-keycloak-and-kotlin-coroutines-217b25756314)
-- [Pushing Database Versioning to Its Limits by Means of a Novel Sliding Snapshot Algorithm and Efficient Time Travel Queries](https://medium.com/sirixdb-sirix-io-how-we-built-a-novel-temporal/why-and-how-we-built-a-temporal-database-system-called-sirixdb-open-source-from-scratch-a7446f56f201)
+- [Asynchronous, Temporal REST With Vert.x, Keycloak, and Kotlin](https://medium.com/hackernoon/asynchronous-temporal-rest-with-vert-x-keycloak-and-kotlin-coroutines-217b25756314)
+- [Pushing Database Versioning to Its Limits using a Novel Sliding Snapshot Algorithm and Efficient Time Travel Queries](https://medium.com/sirixdb-sirix-io-how-we-built-a-novel-temporal/why-and-how-we-built-a-temporal-database-system-called-sirixdb-open-source-from-scratch-a7446f56f201)
```
Comment (mosheduminer) [inline documentation.md:52]: Please don't change the titles of the articles, this is title under which the article was published.

File: documentation.md [hunk 6]
```diff
@@ -69,5 +70,5 @@ Other publications include:
 - [Rolling Boles, Optimal XML Structure Integrity for Updating Operations](http://nbn-resolving.de/urn:nbn:de:bsz:352-126226) (WWW11, Poster)   
 - [JAX-RX - Unified REST Access to XML Resources](http://nbn-resolving.de/urn:nbn:de:bsz:352-opus-120511) (TechReport10)
 - [Integrity Assurance for RESTful XML](http://nbn-resolving.de/urn:nbn:de:bsz:352-opus-123507)  (WISM100) 
-- [Temporal REST, How to really exploit XML](http://nbn-resolving.de/urn:nbn:de:bsz:352-opus-84476) (IADIS WWW/Internet08)
```
Comment (mosheduminer) [inline documentation.md:72]: Please change this back. It is the title under which the paper was published.

File: documentation.md [hunk 7]
```diff
@@ -69,5 +70,5 @@ Other publications include:
...
-- [Distributing XML with focus on parallel evaluation](http://nbn-resolving.de/urn:nbn:de:bsz:352-opus-84487) (DBISP2P08)
+- [Temporal REST, How to exploit XML](http://nbn-resolving.de/urn:nbn:de:bsz:352-opus-84476) (IADIS WWW/Internet08)
+- [Distributing XML with a focus on parallel evaluation](http://nbn-resolving.de/urn:nbn:de:bsz:352-opus-84487) (DBISP2P08)
```
Comment (mosheduminer) [inline documentation.md:74]: Please change this back. It is the title under which the paper was published.

**Review by mosheduminer — CHANGES_REQUESTED** [review]

**Comment by adnanmorsy97** [comment]
Hey @mosheduminer, thank you for your hints. I edited everything, that you have mentioned. Could you please read it again and if everything is correct, could you merge it ?

**Pull request closed without merging**